src/Tribe/Subscribe/Purchase.php

```diff
@@ -52,6 +52,10 @@ public function connect( $attendee_id, $post_id, $order, $product_id ) {
 				'property' => 'lastname',
 				'value'    => $order->get_billing_last_name(),
 			],
+			[
+				'property' => 'newcustomproperty',
+				'value'    => 'Custom Property Test',
```

Comment:
Temporary, right? Should you add a todo for yourself to come back to this?

Reply:
I removed it. It was just a test.